mariannaFervenza/Quiz/Q1.py

```diff
@@ -0,0 +1,40 @@
+# Given an array of positive numbers and a positive number ‘k,’ find the maximum sum of any contiguous subarray of #size ‘k’.
+
+
+# So we are given an array of k = [positive numbers]
+# and we are given a variable k = "integer that is the size of the array"
+# We need to find the max sum of of this subarrays with size k
+
+def maxSub(arr, k):
+    length = len(arr)
+    index = 0
+
+    if length == 0:
+        return 0
+
+    result = 0
+    maxSum = result
+    while index >= k:
+
+        maxSum += arr[index] - arr[index - k]
+        result = max(result, maxSum)
+
+        return result
+
+    if length < k:
```

Comment:
should this edge case be presented at the top of the program for an early return?